### DISCUSSION TITLE
development/2018/08/24/data-visualization-in-python/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#99] [Comments]
#  Python에서 데이터 시각화하는 다양한 방법

https://zzsza.github.io/development/2018/08/24/data-visualization-in-python/

**Comment by zzsza** [comment]
<p>정리 감사합니다</p>

>_This comment was made by Disqus. 2018-09-06 05:28:53 김지우_

**Comment by zzsza** [comment]
<p>감사합니다:)</p>

>_This comment was made by Disqus. 2018-09-10 03:09:45 Yarmini_

**Comment by zzsza** [comment]
<p>잘 정리되어있네요. 잘봤습니다. :)</p>

>_This comment was made by Disqus. 2018-09-18 11:49:38 개발자스럽다_

**Comment by zzsza** [comment]
<p>혹시 이렇게 만든 그래프를 PPT 등에 올리고 싶으면 어떻게 해야 하나요 ㅠㅠㅠ</p>

>_This comment was made by Disqus. 2019-03-26 11:38:58 김수영_

**Comment by zzsza** [comment]
<p>잘 정리된 자료 고맙습니다. Echarts도 파이썬에서 쓸 수 있다니 좋네요.</p>

>_This comment was made by Disqus. 2019-03-30 08:35:27 HyunYoung Go_

**Comment by zzsza** [comment]
<p>pyecharts 에서 오른쪽에 있는 중국어 지우고 싶은데 가능한가요??</p>

>_This comment was made by Disqus. 2019-07-02 03:15:18 최현준_

**Comment by zzsza** [comment]
<p>tips = sns.load_dataset("tips") 실행시,</p><p>OSError: Initializing from file failed 발생하는 이유가 뭘까요?</p>

>_This comment was made by Disqus. 2019-07-25 06:01:38 ksw_

**Comment by zzsza** [comment]
<p>정말 많이 배워갑니다.<br>감사드립니다.<br>건승하시길 바랍니다.</p>

>_This comment was made by Disqus. 2019-08-10 11:41:36 블랙로즈_

**Comment by zzsza** [comment]
<p>pyecharts 라이브러리가 궁굼합니다.<br>pycharm  에서 어떻게 실행해야 하는지요.<br>코드 실행하면 그래프가 아무것두 않뜨더라구 참고로 render.html파일이 생성되어서 이것도 실행<br>해보았지만 빈페이지만 뜹니다. 이렇게 좋은 자료 공유해주셔서 감사 합니다. ^^</p>

>_This comment was made by Disqus. 2019-10-22 09:18:34 방성수_

**Comment by zzsza** [comment]
<p>안녕하세요 정말 좋은 자료 공유 감사합니다! 혹시 plotly 사용 중 유료 구독에 대한 에러 문자를 보신 적은 없으신가요? <br>plotly로 간단한 개발을 하고 있던 와중에 plotly.plotly는 더이상 사용하지 않는 모듈이라고 뜨고 chart_studio.plotly를 권장하길래 import하여 사용하려 하였으나 제 계정을 갱신하라는 에러 문구를 보아서 문자 남깁니다.</p>

>_This comment was made by Disqus. 2019-10-26 07:36:11 SUZ Yang_

**Comment by zzsza** [comment]
<p>혹시 타이타닉 아시나요? 거기서 생존자 예측할때 변수가 많은거처럼 제가 변수가 7개있으면 변수사이의 관계를 보고 싶은데.... 7D plot같은건 어케 해야할까요? 원하는건 7개변수와 final answer의 관계지만 상관관계를 data기반 자세히 들여다보고 싶습니다.</p>

>_This comment was made by Disqus. 2020-03-13 07:56:04 INNO TECH_

**Comment by zzsza** [comment]
<p>플로틀리 처음 시도해봤는데 포스팅이 큰 도움 되었습니다.</p><p>다만, 패키지 업데이트로 인해 이제는 import plotly 하고 iplot 을 바로 하는 게 아니라<br>from chart_studio.plotly import plot, iplot<br>라는 chart_studio 패키지를 써야한다고 공지되었습니다.</p><p>안내 링크 :<br><a href="https://plotly.com/python/v4-migration/" rel="nofollow noopener" title="https://plotly.com/python/v4-migration/">https://plotly.com/python/v...</a></p>

>_This comment was made by Disqus. 2020-04-14 12:53:17 김문과의데이터_

**Comment by zzsza** [comment]
<p>안녕하세요 ㅠㅠ 게시글 너무 잘 읽었습니다. 혹시 pyecharts 모듈로 제작한 그래프는 어떻게 save하나요??  또한 동적인 그래프라서 어떤 형식으로 저장되는지 궁금합니다.</p>

>_This comment was made by Disqus. 2020-11-01 04:42:13 지영석_

**Comment by zzsza** [comment]
<p>Gauge는  3단계로밖에 구분이 안되나요? 25%씩 해서 4단계로 나누고싶은데 코드 여기저기 찾아봐도 찾기가 쉽지가 않습니다.</p>

>_This comment was made by Disqus. 2021-10-25 01:12:03 김재현_

**Comment by fecu89** [comment]
어디 광고 없나요..? 클릭해드리고 싶네요. 잘 배우고 갑니다.

**Reply by fecu89** [reply]
맨 위에 있었군요 ㅎㅎ 열심히 클릭하고 가요
